services.yaml

```diff
@@ -4029,6 +4060,80 @@ definitions:
             example: "http://hooks.company.com/overops1/P1/hook"
       #jira_settings: <- TBD
       #udf_settings <- TBD
+  ViewAlertsSummary:
+    description: basic alerts info for a single view
+    type: object
+    properties:
+      view_id:
+        type: string
+        description: View ID
+        example: P12
+      view:
+        type: string
+        description: View Name
+        example: "API"
+      categories:
+        type: array
+        description: Categories Names
+        items:
+          type: string
+        example: "Applications"
+      alerts:
+        type: object
+        properties:
+          global:
+            type: boolean
+            description: Indicates alerts which are global, i.e. will affect all users that share this view.
+            example: false
+          alert_on_new:
+            type: "boolean"
+            description: Indicates alerts which are sent on new events
+            example: true
+          anomaly_alert:
+            type: object
+            properties:
+              alert_on_anomaly:
+                type: "boolean"
+                example: true
+              anomaly_function:
```

Comment:
Why is this not a dedicated object (2)

Reply:
added a dedicated object. However, since the example will be given for the unified object, the channel and anomaly function objects will be represented by the same example and it can be confusing for the user. wdyt?